_posts/2023-06-16-InauguralNewsletter.md

```diff
@@ -1,9 +1,12 @@
 ---
-layout: doc
-title: Inaugural issue of the OBO Foundry Newsletter
+layout: post
+title: "OBO Inaugural Newsletter June 2023"
+date: 2023-06-16
+categories: update
+summary: "The inaugural edition of our newly formed OBO Newsletter."
 ---
 
-## Welcome to the inaugural issue of the OBO Foundry Newsletter -  Celebrating our 20th anniversary
+## Welcome to the inaugural issue of the OBO Foundry Newsletter -  Celebrating our 20th anniversary (June 2023)
```

Comment:
put this whole header into the title and remove the title. Then the first paragraph becomes the short text displayed on the blog roll

Reply:
Ok done, thank you!